spec/acceptance/node_classification_spec.rb

```diff
@@ -0,0 +1,128 @@
+require 'spec_helper_acceptance'
+require 'json'
+
+describe 'node classification' do
+  let(:params) do
+    servicenow_config = servicenow_instance.bolt_config['remote']
+
+    {
+      instance: servicenow_instance.uri,
+      user: servicenow_config['user'],
+      password: servicenow_config['password'],
+    }
+  end
+
+  test_environment = 'node_classification_tests'
+  test_environment_code_dir = "/etc/puppetlabs/code/environments/#{test_environment}"
```

Comment:
Assigning variables outside of an rspec construct that controls when the code is called can lead to thorny scope issues. Can these be turned into a `let` or something like that?

Reply:
No can do unfortunately. `let` is only allowed for `it`/`before(:each)` blocks (something about "per-test state"). Made them constants instead.

Reply:
Added a comment explaining this too.